pipelines/edx-sysadmin-publish.yml

```diff
@@ -0,0 +1,56 @@
+---
+resource_types:
+- name: pypi
+  type: docker-image
+  source:
+    repository: cfplatformeng/concourse-pypi-resource
+    tag: latest
+
+resources:
+    - name: edx-sysadmin-repo
+      type: git
+      source:
+          uri: https://github.com/mitodl/edx-sysadmin.git
+          branch: ((release))
+
+    - name: edx-sysadmin-pypi
+      type: pypi
+      source:
+        name: edx-sysadmin
+        packaging: any
+        repository:
+          username: ((username))
```

Comment:
We should change these vars to be something like `pypi_creds.username` and `pypi_creds.password` so that we can put them under a single key in Vault that Concourse can pull dynamically.